pkg/azuredevops/pipelines/templatesParams.go

```diff
@@ -149,8 +149,8 @@ func (p OCIImageBuilderTemplateParams) Validate() error {
 	if jobType, ok = p["JobType"]; !ok {
 		return ErrRequiredParamNotSet("JobType")
 	}
-	if jobType != "presubmit" && jobType != "postsubmit" && jobType != "on-demand" {
-		return fmt.Errorf("JobType must be either presubmit, postsubmit or on-demand, got: %s", jobType)
+	if jobType != "presubmit" && jobType != "postsubmit" && jobType != "workflow_dispatch" {
+		return fmt.Errorf("JobType must be either presubmit, postsubmit or workflow_dispatch, got: %s", jobType)
```

Comment:
```suggestion
	if !slices.Contains([]string{"presubmit", "postsubmit", "workflow_dispatch"}, jobType) {
		return fmt.Errorf("JobType must be either presubmit, postsubmit or workflow_dispatch, got: %s", jobType)
```

It would be sligtly more readable with that approach, but personally I prefer extracting such slice as a static defined slice in e.g. config package? So we keep all supported event list in one place.

To be honest it's only small consideration to think about, but it's not required for now.